docs/dev-guide/guide.md

```diff
@@ -544,14 +544,15 @@ await updateEnrollmentStatus({
 
 In this example, instructors are typically allowed to read the enrollment record for students, but for certain actions, like joining a course instance, they need to be authorized as a student. The model function would note that the `requestedRole` parameter is `'Student'`, but the current user is an instructor, so it would throw an error.
 
-In some cases, you may not have access to `authzData`, e.g. if you are pulling data from a queue, or deep in internal code. In this case, you can use the `dangerousFullAuthzForTesting` function to build a dummy `authzData` object that allows you to perform the action as the system. NOTE: We need to revisit this concept as we build out this pattern.
+In some cases, you may not have access to `authzData`, e.g. if you are pulling data from a queue, or deep in internal code. In this case, you can use the `dangerousFullSystemAuthz` function to build a dummy `authzData` object that allows you to perform the action as the system. NOTE: We need to revisit this concept as we build out this pattern.
```

Comment:
Do we still need to revisit? Or do you think this `dangerousFullSystemAuthz()` thing is a sensible place to have ended up?

Reply:
We confirmed.